tools/protodoc/generate_empty.py

```diff
@@ -7,25 +7,25 @@
 import sys
 import tarfile
 
-import protodoc as protodoc
+import protodoc as _protodoc
```

Comment:
Why underscore? This feels like some internal name. Can we use something like `protodoc_import` or `pd` etc?

Reply:
this _is_ a common thing to do - i understand and agree with your previous objection regarding local vars - but this is pretty idiomatic - ie using an underscore prefix to disambiguate a namespace clashing imported object

if its a blocker i can use `pd` - i have never seen an import disambiguated with `x_import` before so i would rather not do that